src/coreclr/pal/src/libunwind/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ if(CLR_CMAKE_HOST_UNIX)
       set(arch x86_64)
     elseif(CLR_CMAKE_HOST_ARCH_ARM64)
       set(arch aarch64)
+    elseif(CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
Unless this results in build breakage, I would move changes from this file to https://github.com/dotnet/runtime/pull/62887 that way each PR would be self-contained.

Reply:
Thanks.
Had moved.